### PR TITLE
ADAPT-000: Change static graphql query from ID to slug for multi environment.

### PR DESCRIPTION
**Pull request by sherakama**
# READY FOR REVIEW

# Summary
- Changes the graphql lookup from ID to full slug and protects variables
- Builds were failing on dev when I used the CLI tool to sync content back down stream and the UUIDs changed

# Review By (Date)
- Whenever

# Criticality
- Low

# Review Tasks

1. Review code
2. Review preview build to see that modal content still appears
3. Pull down branch to local
4. Rebuild on local with API key set to DEV storyblok space
5. Validate that the content in the modal still appears
6. Change the graphql query so it fails
7. Rebuild local environment
8. Validate build still works and page renders but content is missing

## Front End Validation
- [ ] Is the markup using the appropriate semantic tags and passes HTML validation?
- [ ] Cross-browser testing has been performed?
- [ ] Automated accessibility scans performed?
- [ ] Manual accessibility tests performed?
- [ ] Design is approved by @ user?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket(s)
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

File: src/components/search/searchModal.js [hunk 1]
```diff
@@ -18,10 +22,14 @@ const SearchModal = ({ isOpen, setIsOpen, onClose }) => {
 
   let story;
   let content;
+  let introduction;
+  let emptySearchMessage;
 
-  if (data) {
+  if (data && data.storyblokEntry && data.storyblokEntry.content) {
```
Comment (mattanglin) [inline src/components/search/searchModal.js:28]: Random Typescript plug:
In TS this condition would be `if (data?.storyBlokEntry?.content) {`!

**Review by mattanglin — APPROVED** [review]
👍🏼

File: src/components/search/searchModal.js (hunk 1 above)
Comment (sherakama) [reply src/components/search/searchModal.js:28]: Probably should have used it.

Comment (sherakama) [reply src/components/search/searchModal.js]: Let's try this out.

**Pull request merged**